admin/tracking/class-tracking-default-data.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @package WPSEO\Admin\Tracking
+ */
+
+/**
+ * Represents the default data.
+ */
+class WPSEO_Tracking_Default_Data implements WPSEO_Collection {
+
+	/**
+	 * Returns the collection data.
+	 *
+	 * @return array The collection data.
+	 */
+	public function get() {
+		return array(
+			'name'         => $this->get_user_info( 'name' ),
+			'email'        => $this->get_user_info( 'email' ),
+			'@timestamp'   => date( 'd-m-Y H:i:s' ),
+			'wpVersion'    => $this->get_wordpress_version(),
+			'isMultisite'  => is_multisite(),
+			'siteLanguage' => get_bloginfo( 'language' ),
+		);
+	}
+
+	/**
+	 * Returns info about the current user.
+	 *
+	 * @param string $what What to retrieve, defaults to name.
+	 *
+	 * @return string
+	 */
+	protected function get_user_info( $what = 'name' ) {
+		$current_user = wp_get_current_user();
+		switch ( $what ) {
+			case 'email':
+				return $current_user->user_email;
+			case 'name':
+			default:
```

Comment:
Default should not return 'name'.
When the function is called without an argument 'name' is used anyway, so if we are not catching the argument, it is given and should not be returning something else.